spec/models/legal_aid_application_spec.rb

```diff
@@ -173,4 +173,64 @@
       end
     end
   end
+
+  describe '#own_home?' do
+    context 'legal_aid_application.own_home is nil' do
+      before { legal_aid_application.update!(own_home: nil) }
+      it 'returns false' do
+        expect(legal_aid_application.own_home?).to eq(false)
+      end
+    end
+
+    context 'legal_aid_application.own_home is "no"' do
+      before { legal_aid_application.update!(own_home: 'no') }
```

Comment:
FYI - as `own_home` is an enum you can do this: `legal_aid_application.own_home_no!` to set `own_home` as 'no'

However, i'd suggest a better way of doing this is to use the existing traits (less database calls) to define `legal_aid_application`:
```ruby
context 'legal_aid_application.own_home is "no"' do
  let(:legal_aid_application) { create :legal_aid_application, :without_own_home }
  it 'returns false' do
    expect(legal_aid_application.own_home?).to eq(false)
  end
end
```